src/utils/misc.cc

```diff
@@ -698,3 +708,84 @@ void packagekit_suggest_quit()
   ExternalProgram pkcall( argv );
   pkcall.close();
 }
+
+
+bool iType( const ui::Selectable::constPtr & sel_r )
```

Comment:
using ui::Selectable::constPtr here just works because it included from some other header, we are forward declaring ui::Selectable at the top but not constPtr

Reply:
I'll add the Selectable.h include